src/EnsembleKalmanInversion.jl

```diff
@@ -46,14 +46,13 @@ covariance matrix no greater than cov_threshold.
 function find_ekp_stepsize(
     ekp::EnsembleKalmanProcess{FT, IT, Inversion},
     g::AbstractMatrix{FT};
-    cov_threshold::FT = 0.01,
+    cov_threshold::Real = 0.01,
 ) where {FT, IT}
+    @assert cov_threshold > -eps(FT) "The limiting covariance reduction threshold cannot be negative."
```

Comment:
`eps(Real)`?

Reply:
`eps(Real)` does not exist, unfortunately, since it does not specify the arithmetic precision.